specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json

```diff
@@ -187,17 +187,7 @@
         },
         "catalogPricingTier": {
           "description": "The pricing tier for the catalog database. The valid values could be found in https://azure.microsoft.com/en-us/pricing/details/sql-database/",
-          "type": "string",
-          "enum": [
-            "Basic",
-            "Standard",
-            "Premium",
-            "PremiumRS"
-          ],
-          "x-ms-enum": {
-            "name": "IntegrationRuntimeSsisCatalogPricingTier",
-            "modelAsString": true
-          }
+          "type": "string"
```

Comment:
This is not allowed though. This is a stable API version so no breaking changes can be introduced.

Reply:
@jianghaolu I revoked the changes in stable API version.